src/lib/holocene/accordion/accordion.svelte

```diff
@@ -56,11 +56,12 @@
       on:click={toggleAccordion}
     >
       <div class="space-between flex w-full flex-row items-center">
-        <h3 class="flex w-full items-center gap-2">
+        <h3 class="flex min-w-fit items-center gap-2">
```

Comment:
The change to `min-w-fit` here from `w-full` means that the chevron icon on the `Accordion` isn't positioned correctly when there isn't a "description" slot 

| Before | After |
|-|-|
|<img width="742" alt="Screenshot 2024-09-16 at 3 21 59 PM" src="https://github.com/user-attachments/assets/ca3ea2cc-1c5b-47ae-851b-c50eeae55497">|<img width="731" alt="Screenshot 2024-09-16 at 3 20 20 PM" src="https://github.com/user-attachments/assets/9047f3a7-60b4-406d-a0fc-c65daa7f0234">|

Reply:
Good call, changed it back.